fundamental-concepts/data-structures-stdlib/index.json

```diff
@@ -0,0 +1,44 @@
+{
+  "title": "Data types and the DAML standard library",
+  "description": "Learn about records, sums, sets, and maps and other essential tools of the DAML standard library.",
+  "difficulty": "Beginner",
+  "time": "30 minutes",
+  "details": {
+    "assets": {
+        "host01": [
+        ]
+    },
+    "steps": [
+      {"text": "1_records.md"},
```

Comment:
I think we should add answers to any steps with quizzes. Otherwise people just get stuck. See here to see how: https://github.com/katacoda/scenario-examples/tree/master/verified-steps

Reply:
does this work with the quiz syntax provided by katacoda?

Reply:
it sadly doesn't, but we requested this as a feature from the katacoda team. for now, all solutions are provided at the bottom so people are not getting stuck.